asp.tf

```diff
@@ -1,15 +1,16 @@
-locals {
-  ase_name = "core-compute-${var.env}"
-}
-
-module "appServicePlan" {
-  source              = "git@github.com:hmcts/cnp-module-app-service-plan?ref=master"
-  location            = "${var.location}"
-  env                 = "${var.env}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
-  asp_sku_size        = "${var.asp_sku_size}"
-  asp_capacity        = "${var.asp_capacity}"
-  asp_name            = "${var.product}"
-  ase_name            = "${local.ase_name}"
-  tag_list            = "${var.common_tags}"
-}
+//// commented out to stop recreating legacy app service plans
+//locals {
+//  ase_name = "core-compute-${var.env}"
+//}
+//
+//module "appServicePlan" {
+//  source              = "git@github.com:hmcts/cnp-module-app-service-plan?ref=master"
+//  location            = "${var.location}"
+//  env                 = "${var.env}"
+//  resource_group_name = "${azurerm_resource_group.rg.name}"
+//  asp_sku_size        = "${var.asp_sku_size}"
+//  asp_capacity        = "${var.asp_capacity}"
+//  asp_name            = "${var.product}"
+//  ase_name            = "${local.ase_name}"
+//  tag_list            = "${var.common_tags}"
+//}
```

Comment:
Comment out ASP so that it doesn't build it in Azure as we're moving to AKS

Reply:
Still needed  for FE apps?

Reply:
As per https://github.com/hmcts/div-shared-infrastructure/pull/53#discussion_r329478669